### PR TITLE
Fix e2e tests

### PR DESCRIPTION
**Pull request by kdoh**
Makes the query builder tutorial tests a little less fragile by using IDs for images (they would fail on a retina screen cause of the @2x file name). Also updates the QB tutorial to account for the new date picker widget and updates the spec accordingly.

### CODE REVIEW

**Comment by kdoh** [comment]
@metabase/core-developers  (particularly @tlrobinson) let me know what you think of this.

File: frontend/src/metabase/query_builder/components/filters/pickers/DatePicker.jsx [hunk 1]
```diff
@@ -62,7 +62,7 @@ class CurrentPicker extends Component {
 }
 
 
-const getIntervals = ([op, field, value, unit]) => op === "TIME_INTERVAL" && typeof value === "number" ? Math.abs(value) : 1;
+const getIntervals = ([op, field, value, unit]) => op === "TIME_INTERVAL" && typeof value === "number" ? Math.abs(value) : 30;
```
Comment (tlrobinson) [inline frontend/src/metabase/query_builder/components/filters/pickers/DatePicker.jsx]: I assume 30 makes sense because the default interval is "days"?

**Review by tlrobinson — APPROVED** [review]
LGTM.

File: frontend/src/metabase/query_builder/components/filters/pickers/DatePicker.jsx (hunk 1 above)
Comment (kdoh) [inline frontend/src/metabase/query_builder/components/filters/pickers/DatePicker.jsx]: Yeah that was the idea.

**Pull request merged**